tests/Doctrine/Tests/Models/CMS/CmsAddress.php

```diff
@@ -91,7 +91,7 @@ class CmsAddress
 
     /**
      * @OneToOne(targetEntity="CmsUser", inversedBy="address")
-     * @JoinColumn(referencedColumnName="id")
+     * @JoinColumn(referencedColumnName="id", unique=false)
```

Comment:
That's more or less true, but it doesn't mean that this isn't something that a user of the library might want to accomplish.  This is a style that seems to be asked for in more than one place, including in Hibernate: https://hibernate.atlassian.net/browse/HHH-4815  https://hibernate.atlassian.net/browse/HHH-4726

If you find it might be useful, then merge it.   It's the simplest way of getting this to work, more or less, that I could find.